cnf-certification-test/lifecycle/suite.go

```diff
@@ -147,6 +148,12 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
 		testPodPersistentVolumeReclaimPolicy(&env)
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestContainersImageTag)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testContainersImageTag(&env)
```

Comment:
Add `testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Containers)` before the call to the func.